src/main/java/core/basesyntax/service/operation/BalanceOperationHandlerImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandlerImpl implements OperationHandler {
+    private FruitTransactionDao fruitTransactionDao = new FruitTransactionDaoImpl();
```

Comment:
let's use FruitTransactionDao only in FruitTransactionService. You can create methods there to interact with it and call them in places where you need it